01-node-tutorial/answers/02-globals.js

```diff
@@ -0,0 +1,13 @@
+// GLOBALS -NO WINDOW!!!!
+
+// _dirname - path to current directory 
+// _filename - file name  
+// require - function to use modules (commonJS)
+// module - info about current module (file)
+// process - info about where the program is being executed 
+
+
```

Comment:
Set environment variable and print in console to see if it's working. 
Eg. VAR_NAME = "Hello", set in terminal.
And using process module to print.